study-builder/studies/shared/family-history/family-history-maternal-grandfather.conf

```diff
@@ -0,0 +1,7 @@
+{
+  include required("snippets/family-history-grandparent.conf"),
+  activityCode="FAMILY_HISTORY_MATERNAL_GRANDFATHER"
+  NAME_EN="Grandfather (Your mother's side)"
+  TITLE_EN="Grandfather (Your mother's side)"
+  memberPrefix="MATERNAL_GRANDFATHER"
```

Comment:
If I'm understanding the conventions, this should be `MEMBER_PREFIX`?

Reply:
Was the first variable I created to facilitate sharing, but was not clear on what convention to use.
35299766